docs/source/content/create.drip.rst

```diff
@@ -16,11 +16,18 @@ On the other hand, when you click in the ``FIELD NAME OF USER`` input, you will
 
 Here you can relate the Drip to the corresponding ``Campaign``. Grouping several drips under a campaign.
 
+Resend drip field
+-----------------
+
+The default behaviour of Drip library is that an email could only be sended to one user every time.
+Now with the field ``Can resend drip`` you could be able to send the same drip email to one user multiple times. Every time the user matches the conditions, it will be sended.
```

Comment:
```suggestion
Now with the field ``Can resend drip`` you could be able to send the same drip email to one user multiple times. Every time the user matches the conditions, it will receive the email.
```

Reply:
I already apply this suggestion!